kali/ansible/tasks/vnc.yml

```diff
@@ -0,0 +1,47 @@
+---
+- name: Ensure sysconfig directory
+  file:
+    path: /etc/sysconfig
+    state: directory
+
+- name: Set preferred display manager
+  copy:
+    content: "PREFERRED=/usr/bin/xcfe4-session"
+    dest: /etc/sysconfig/desktop
+
+- name: Set VNC password
+  become: yes
+  become_user: kali
+  expect:
+    command: vncpasswd
+    responses:
+      Password:: "mypass\r"
+      Verify:: "mypass\r"
```

Comment:
TODO: Decide how to accept this parameter into the playbook.

Reply:
This password requires an SSH local port forward to be able to connect so I actually think a default password hard-coded here is fine.